src/WorldState.Data/Models/Invasion.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Newtonsoft.Json.Serialization;
+
+using WorldState.Data.Enums;
+
+namespace WorldState.Data.Models
+{
+    [JsonObject(NamingStrategyType = typeof(CamelCaseNamingStrategy), ItemNullValueHandling = NullValueHandling.Ignore)]
+    public class Invasion
+    {
+        [JsonProperty]
+        public String Id { get; private set; }
+
+        [JsonProperty]
+        public DateTimeOffset Activation { get; private set; }
+
+        [JsonProperty]
+        public String Node { get; private set; }
+
+        [JsonProperty("desc")]
+        public String Description { get; private set; }
+
+        public MissionReward AttackerReward { get; private set; }
```

Comment:
This property needs the `[JsonProperty]` attribute or Json.NET will fail to deserialize it.